packages/optimize/ts/AEPOptimize.ts

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { EventSubscription, NativeModules } from 'react-native';
+import { NativeEventEmitter } from 'react-native';
+import Proposition from './models/Proposition';
+import DecisionScope from './models/DecisionScope';
+import { AdobeCallback }  from './models/AdobeCallback';
+
+const RCTAEPOptimize = NativeModules.AEPOptimize;
+
+declare var onPropositionUpdateSubscription: EventSubscription;
+
+var onPropositionUpdateSubscription: EventSubscription;
+
+/**
+* Public APIs for Optimize extension
+*/
+const AEPOptimize = {
```

Comment:
Should we also define an interface for Optimize APIs to keep the consistency with others https://github.com/adobe/aepsdk-react-native/blob/b03cfbc8bc82b0efb4085bd659a00ce70b67be21/packages/edge/ts/Edge.ts#L17?

Reply:
About the module/extension name, I think we'd better remove AEP prefix to follow the naming conventions as other extensions: https://github.com/adobe/aepsdk-react-native/blob/b03cfbc8bc82b0efb4085bd659a00ce70b67be21/packages/edge/ts/Edge.ts#L26.